tensorflow/lite/micro/micro_interpreter_graph.cc

```diff
@@ -224,12 +224,10 @@ TfLiteStatus MicroInterpreterGraph::InvokeSubgraph(int subgraph_idx) {
     // prepare for the next call.
     allocator_->ResetTempAllocations();
 
-    if (invoke_status == kTfLiteError) {
+    if (invoke_status != kTfLiteOk) {
       MicroPrintf("Node %s (number %d) failed to invoke with status %d",
                   OpNameFromRegistration(registration), current_operator_index_,
                   invoke_status);
```

Comment:
@andresovela 

This will cause extraneous debug output for the `kTfLiteAbort` status code.  The `kTfLiteAbort` status code is used by the circular buffer kernel, the layer-by-layer debug tool, and the Generic Benchmark application.

I would suggest adding another `if` statement at line 228 to enable debug output only if the status code is not `kTfLiteAbort`.

Reply:
I wanted to fix this but there's no such thing as `kTfLiteAbort` it seems:

https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/core/c/c_api_types.h#L74-L120

Reply:
Ah I found it https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/micro_context.h#L29-L30

A suspicious cast if you ask me, there should be better documentation on why that exists.